inquire/src/prompts/path_select/modes.rs

```diff
@@ -0,0 +1,50 @@
+//! Prompt modes 
+//! 
+//! Everything here should be exposed to the (library) user
+use strum::{Display, EnumCount, FromRepr};
+
+
+/// Different path selection modes specify what the user can choose
+#[derive(Clone, Eq, PartialEq)]
+pub enum PathSelectionMode<'a> {
+    /// The user may pick a directory.
+    Directory,
+    /// The user may pick a file with the given (optional) extension.
+    File(Option<&'a str>),
+    /// The user can set gitignore rules from a file 
+    /// The user may pick multiple paths
+    Multiple(Vec<PathSelectionMode<'a>>),
+}
+
+impl<'a> Default for PathSelectionMode<'a> {
+    fn default() -> Self {
+        Self::Directory
```

Comment:
I think defaulting to File makes more sense here, specially when we think that picking directories was barely supported on browsers years ago

Reply:
No problem